static/index.html

```diff
@@ -4,7 +4,34 @@
         <meta charset="utf-8">
         <meta content="width=device-width, initial-scale=1" name="viewport">
         <title>ankerctl</title>
-        <link href="/static/vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css" rel="stylesheet">
+        <link href="{{ url_for('static', filename='vendor/bootstrap-5.3.0-alpha3-dist/css/bootstrap.min.css')}}" rel="stylesheet">
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css" rel="stylesheet">
```

Comment:
I was concerned about this, I may find an alternative for icons because pulling in all of the icons from bootstrap may be too large (IE several hundred megs just for images).

Reply:
I just checked the bootstrap release zip. The (uncompressed) size of the css + font versions is about 300KB, so that seems small enough to include?

Reply:
font doesn't include icons

Reply:
The font from `bootstrap-icons` doesn't include icons?

Reply:
I'll double check and pull it in if that is the case :)

Reply:
✅